### PR TITLE
DOC-3732 Research Guide: restructure course team events

### PR DESCRIPTION
**Pull request by catong**
## [DOC-3732](https://openedx.atlassian.net/browse/DOC-3732)
Add organization to the existing Course Team Events section of the edX Research Guide:

dd alphabetical TOC of categories after the introduction; group/move events into appropriate categories. Add mini TOC for each category. Where possible, for events without any description, add a description. 

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @stroilova 
- [x] Doc team review (sanity check/copy edit): @edx/doc 
- [ ] Product review: @sstack22 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### HTML DRAFT OUTPUT FOR REVIEWERS

- [x] http://draft-researchguide-courseteamevents.readthedocs.io/en/latest/internal_data_formats/tracking_logs/course_team_event_types.html#course-team-events
- [x] http://draft-researchguide-courseteamevents.readthedocs.io/en/latest/internal_data_formats/event_list.html

(The currently published version can be found here: http://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/course_team_event_types.html#course-team-events)

### Post-review
- [ ] Squash commits

### CODE REVIEW

**Comment by stroilova** [comment]
Thank you @catong ! our research guide is getting cleaner :)

👍

File: en_us/data/source/internal_data_formats/tracking_logs/course_team_event_types.rst [hunk 1]
```diff
@@ -12,26 +12,119 @@ For a list of events generated by learner interactions in the LMS other than on
...
+``edx.cohort.creation_requested``
+=================================
+
+When a course team member manually creates a cohort on the **Cohorts** page of
+the instructor dashboard, the server emits an
+``edx.cohort.creation_requested`` event.
+
+**Event Source**: Server
+
+**History** Added 7 Oct 2014.
```
Comment (debchatigny) [inline en_us/data/source/internal_data_formats/tracking_logs/course_team_event_types.rst]: Insert a colon after **History**.

File: en_us/data/source/internal_data_formats/tracking_logs/course_team_event_types.rst [hunk 2]
```diff
@@ -12,26 +12,119 @@ For a list of events generated by learner interactions in the LMS other than on
...
+When a course team member adds a student to a cohort on the **Cohorts** page
+of the instructor dashboard, the server emits an
+``edx.cohort.user_add_requested`` event. Course team members can add students
+to a cohort individually, or by uploading a .csv file of student cohort
+assignments.
 
 **Event Source**: Server
 
-For more information about the common fields, see :ref:`common`.
+**History** Added 7 Oct 2014.
```
Comment (debchatigny) [inline en_us/data/source/internal_data_formats/tracking_logs/course_team_event_types.rst]: Insert a colon after **History**. Should there be a period at the end of the date? It is inconsistent with **Component** and **Event Source** in this way.

**Comment by debchatigny** [comment]
Hi Carol,
Everything looks fine to me. I did fine one error (missing text and extra period) but it's outside of a change you've made, so won't let me leave a comment. Here's an image that shows the location in the file: 

image

**Comment by catong** [comment]
@debchatigny Thanks for catching the misformed link text, and the inconsistency with colon after "History". Fixed!

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1552

**Pull request merged**